dataretrieval/nwis.py

```diff
@@ -447,21 +447,43 @@ def get_pmcodes(parameterCd='All', **kwargs):
     Parameters (Additional parameters, if supplied, will be used as query parameters).
     ----------
         parameterCd: string or listlike
+        parameterNm: string
     Returns:
         DataFrame containing the USGS parameter codes and Metadata as tuple
     """
-    payload = {'radio_pm_search' : 'pm_search',
-               'pm_group' : 'All+--+include+all+parameter+groups',
-               'pm_search' : parameterCd,
-               'casrn_search' : None,
-               'srsname_search' : None,
-               'show' :  ['parameter_group_nm', 'casrn', 'srsname','parameter_units', 'parameter_nm'],
-               'format' : 'rdb'}
+    url = 'https://help.waterdata.usgs.gov/code/parameter_cd_nm_query?'
```

Comment:
URLs are configured at the top of the page.  This line is also duplicated at line 466.